src/otx/engine/engine.py

```diff
@@ -301,7 +301,11 @@ def test(
         is_ir_ckpt = Path(str(checkpoint)).suffix in [".xml", ".onnx"]
         if is_ir_ckpt and not isinstance(model, OVModel):
             datamodule = self._auto_configurator.get_ov_datamodule()
-            model = self._auto_configurator.get_ov_model(model_name=str(checkpoint), meta_info=datamodule.meta_info)
+            model = self._auto_configurator.get_ov_model(
```

Comment:
This change does not affect users of the `openvino_model.yaml` configuration.
But in this PR, there are no changes to detection's `openvino_model.yaml`, was that taken into account?

Reply:
I'll try my best to clarify, although I'm not entirely sure I understand your question.

Regarding `openvino_model` recipe, from a user perspective, enabling tiling during training is essential, as some tiling parameters are adjusted within the tile adaptor during training. When exporting the model, one can simply utilize the config.yaml saved post-training for model export, ensuring that all tile configurations are preserved within IR for OV inferencing.

Creating a separate openvino_model recipe for tiling purposes is feasible. However, it's useless if the model hasn't been trained with tiling. In such cases, exporting with this config serves no practical purpose. Furthermore, since tile adaptor dynamically modifies tile size, it poses a challenge to inject these alterations to openvino_model recipe.

Given that tiling must be enabled during both training and testing phases, I believe the optimal approach is to export using the checkpoint config yaml file.